lib/ServerDate.js

```diff
@@ -43,7 +43,37 @@ var
   precision,
   offset,
   target = null,
-  synchronizing = false;
+  synchronizing = false,
+
+  // ServerDate default configuration
+  config = {
```

Comment:
I don't want this extra variable and the following code to merge it.  Please keep the options as they were before.